llarp/dns/dns.hpp

```diff
@@ -1,30 +1,29 @@
 #pragma once
 
+#include <llarp/util/logging.hpp>
```

Comment:
I think IWYU got this one wrong...can't see how this header would need included here.